packages/gpl-core/src/sessionWallet.ts

```diff
@@ -0,0 +1,113 @@
+import * as anchor from "@project-serum/anchor";
+import { Wallet } from "@project-serum/anchor/dist/cjs/provider";
+import { PublicKey, Keypair, Cluster } from "@solana/web3.js";
+import { BN } from "@project-serum/anchor";
+import gpl_session_idl from "./idl/gpl_session.json";
+import { GPLSESSION_PROGRAMS } from "./constants";
+
+export class SessionWallet {
```

Comment:
If this is always gonna be in a node environment, then I think it is better to confirm to how others are implemented. I reckon `SessionWallet` is confusing if it doesn't behave like a wallet.

Reply:
changed the class name to `SessionTokenManager`